public/css/hardware.css

```
@@ -52,7 +52,7 @@ body {
     padding: 0;
   }
   
-  li {
+  #rows {
```

Use a class, not an id.
```suggestion
  .rows {
```